tools/cldr-code/src/main/java/org/unicode/cldr/util/abstracts/AbstractCache.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
```

Comment:
Is this boilerplate comment generated by Eclipse?

Reply:
by netbeans, but yes